### DISCUSSION TITLE
MacOS version UI tuning

### DISCUSSION THREAD

**Discussion by mskvsk** [DeaDBeeF-Player/deadbeef#2895] [Q&A] [answered]
Thank you so much for such an amazing player!

Is there a way to change the font size and type of the playlist?

**Comment by Oleksiy-Yakovenko** [comment, marked as answer]
Hello,

the macOS(cocoa) version of the UI is still not as customizable as the GTK version, but it's slowly getting there.
Maybe I'll get around to adding appearance customizations in 2023 - who knows :)
But right now, you can only change this stuff by editing the code directly.

**Comment by mskvsk** [comment]
Thanks for the quick response and looking forward to it!
